appinventor/components/src/com/google/appinventor/components/runtime/ChartView.java

```diff
@@ -9,7 +9,6 @@
 import android.os.AsyncTask;
 import android.os.Handler;
 import android.os.Looper;
-
```

Comment:
Since this whitespace-only change is the sole modification in this file, please revert the file to match ucr.